Feature: Team mentor (#28)
cilium/team-manager: README.md

        - aanm
        - borkmann
        - joestringer

>         # Optional list of team mentors who will be excluded from team review notifications

Reviewer: There's another GitHub feature for "team review notifications" that will not be changed by this configuration. The intent here is specifically about review assignment.

```suggestion
        # Optional list of team mentors who will not be auto-assigned PRs for review
```
